params/protocol_params.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	GasLimitBoundDivisor uint64 = 1024    // The bound divisor of the gas limit, used in update calculations.
-	MinGasLimit          uint64 = 5000    // Minimum the gas limit may ever be.
-	GenesisGasLimit      uint64 = 4712388 // Gas limit of the Genesis block.
+	GasLimitBoundDivisor uint64 = 1024     // The bound divisor of the gas limit, used in update calculations.
+	MinGasLimit          uint64 = 5000     // Minimum the gas limit may ever be.
+	GenesisGasLimit      uint64 = 4712388  // Gas limit of the Genesis block.
```

Comment:
Is this still used? How is it different from `DefaultGasLimit`?

Reply:
Changing this broke some tests, will investigate further